specs/rm/requirements-management-shapes.html

```diff
@@ -0,0 +1,372 @@
+      }
+
+      var respecConfig = {
+        // === Revision-specific ===
+
+        // the specification's short name with version but no dots
+        shortName: "requirements-management-vocab",
+        specStatus: status,
+        revision: revision.toString().padStart(2, "0"),
+        //publishDate: "2020-xx-xx", //Set date once it is time to publish the PS
+        previousPublishDate: "2019-10-01",
+        previousMaturity: "PSD",
+
+        prevVersion: "https://docs.oasis-open-projects.org/oslc-op/rm/v2.1/psd02/requirements-management-spec.html",
+        latestVersion: oasisBase + "requirements-management-vocab.html",
+
+        thisVersion: thisBase + "requirements-management-vocab.html",
+        latestSpecVersion: "https://open-services.net/spec/rm/latest",
+        edDraftURI: "https://open-services.net/spec/rm/latest-draft",
+
+        // Other parts of multi-part spec
+        additionalArtifacts: [
+          {
+            title: "OSLC Requirements Management Version 2.1. Part 1: Specification",
+            href: "requirements-management-spec.html",
+          },
+          {
+            title: "OSLC Requirements Management Version 2.1. Part 2: Vocabulary",
+            href: "requirements-management-vocab.html",
+          },
+          {
+            title: "OSLC Requirements Management Version 2.1. Part 3: Constraints (this document)",
+            href: "requirements-management-shapes.html",
+          },
+          {
+            title: "OSLC Requirements Management Version 2.1. Part 4: Machine-readable Vocabulary Terms",
+            href: "Requirements-management-vocab.ttl",
+          },
+          {
+            title: "OSLC Requirements Management Version 2.1. Part 5: Machine-readable Constraints",
+            href: "Requirements-management-shapes.ttl",
+          },
+        ],
+
+        relatedWork: [
+          {
+            title: "Open Services for Lifecycle Collaboration Requirements Management Specification Version 2.0",
+            href: "http://open-services.net/bin/view/Main/RmSpecificationV2",
+          },
+        ],
+
+        localBiblio: {
+          OSLCCore2: {
+            title: "OSLC Core 2.0",
+            href: "http://open-services.net/bin/view/Main/OslcCoreSpecification",
+            authors: ["S. Speicher", "D. Johnson"],
+            status: "Finalized",
+            publisher: "http://open-services.net",
+          },
+          OSLCCore3: {
+            title: "OSLC Core 3.0",
+            href: "https://docs.oasis-open.org/oslc-core/oslc-core/v3.0/oslc-core-v3.0-part1-overview.html",
+            authors: ["Jim Amsden", "S. Speicher"],
+            status: "Project Specification",
+            publisher: "OASIS",
+          },
+          OSLCCore3LinkGuidance: {
+            title: "OSLC Core 3.0: Link Guidance",
+            href: "https://raw.githack.com/oslc-op/oslc-specs/master/notes/link-guidance.html",
```

Comment:
we shall stop using any GitHack refs, that was a stopgap measure

Reply:
SO should I just have a relative URL?